WcaOnRails/Gemfile.lock

```diff
@@ -468,7 +474,7 @@ GEM
       thor (>= 0.19.0, < 2.0)
     rainbow (3.0.0)
     raindrops (0.19.1)
-    rake (13.0.3)
+    rake (13.0.6)
```

Comment:
I'd rather not "sideload" dependency updates like this under the radar.

Reply:
Aha! Sorry, totally missed this. I ran `bundle update cookies_eu`, but apparently the right incantation is `bundle update --conservative cookies_eu` (from https://stackoverflow.com/a/23268530/1739415)